src/runtime/crt/common/crt_runtime_api.c

```diff
@@ -112,7 +112,7 @@ int TVMDeviceAllocDataSpaceWithScope(DLDevice dev, int ndim, const int64_t* shap
 
 int TVMDeviceFreeDataSpace(DLDevice dev, void* ptr) { return TVMPlatformMemoryFree(ptr, dev); }
 
-static bool IsContiguous(const DLTensor* arr) {
+TVM_ATTRIBUTE_UNUSED static bool IsContiguous(const DLTensor* arr) {
```

Comment:
Can you clarify why this was needed? was it for eliminating the warning?

Reply:
`IsContiguous()` is only used in an `assert()` below, so in a non-debug build we get an unused function warning